elephant/spectral.py

```diff
@@ -1057,7 +1048,10 @@ def multitaper_coherence(signal_i, signal_j, n_segments=8, len_segment=None,
         Phase lags associated with the magnitude-square coherence estimate
 
     """
-    signals = np.vstack([signal_i, signal_j])
+    if isinstance(signal_i, neo.core.AnalogSignal):
+        signals = signal_i.merge(signal_j)
+    elif isinstance(signal_i, np.ndarray):
+        signals = np.vstack([signal_i, signal_j])
 
```

Comment:
Need a check for the case where `signal_i` is `np.ndarray` and `signal_j` is `neo.AnalogSignal` (or vice versa).

Reply:
Addressed in https://github.com/NeuralEnsemble/elephant/pull/525/commits/5ddd042c88783fdc4203819a36146c746510599d